src/components/workshop-code.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import CardIcon from "@/components/ui/card-icon";
+import { cn } from "@/lib/utils";
+import { useState } from "react";
+import { useToast } from "@/components/ui/use-toast";
+
+export const workshopCodeLiteral = "DKEEH";
```

Comment:
Does this need to be exported? It isn't being used elsewhere. Top-level constants like this can be declared like so:
```ts
const WORKSHOP_CODE_LITERAL = "DKEEH";
// this is now accessible within the rest of the file and is read-only
```

Constants should only be exported when they need to be used within other files. If this is only being used here, it should not be exported.

See [here](https://typescript-eslint.io/rules/naming-convention/) for more info.
> all global-level constants are written in `UPPER_CASE`